src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -442,31 +450,100 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
             return null;
         }
 
+        private bool IsParamsArrayExpanded(SemanticModel semanticModel, SyntaxNode node, SymbolInfo symbolInfo, CancellationToken cancellationToken)
+        {
+            if (symbolInfo.Symbol == null)
+            {
+                return false;
+            }
+
+            int argumentCount;
+            bool lastArgumentHasNameColon;
+            ExpressionSyntax lastArgumentExpression;
+
+            if (node is AttributeSyntax attribute)
+            {
+                argumentCount = attribute.ArgumentList.Arguments.Count;
+                lastArgumentHasNameColon = attribute.ArgumentList.Arguments.LastOrDefault()?.NameColon != null ||
+                    attribute.ArgumentList.Arguments.LastOrDefault()?.NameEquals != null;
```

Comment:
variable name doesn't seem to match initialization logic.  i.e. it says "lastArgHasNameColon" but you check for name-colon and name-equals.  needs comment.

Reply:
Changed to `lastArgumentIsNamed`